src/routes/ConsortiumManager/settings/users/general/PermissionSets/PermissionSetsCompare/PermissionSetsCompareItem.js

```diff
@@ -0,0 +1,135 @@
+import React, {
```

Comment:
Please, take attention how our modules' imports look like:
```
// node_modules imports
import PropTypes from 'prop-types';
...
// ^^^

// @folio/... imports
import { Selection } from '@folio/stripes/components'
...
// ^^^

// local imports
import { Component } from '../components';
...
// ^^^

// code here
```

Reply:
I see what you meant. We should use some kind of Library auto fixer and prettier for such cases.

Reply:
@alisher-epam , alas, I haven't been able to get people on-board with prettier. I thought we did have lint configured to whine about the order of imports from node_modules and local files. Maybe you're not getting a lint warning because this project is using the non-standard acquisitions lint config? acq went rogue years ago, crafting their own rules without telling anybody, and it was too late to rein in by the time we discovered it.